smart-contract/assembly/contracts/deweb-interface.ts

```diff
@@ -136,3 +145,51 @@ export function withdraw(binaryArgs: StaticArray<u8>): void {
 
   transferCoins(Context.caller(), amount);
 }
+
+/**
+ * Deletes a file from the contract storage.
+ * @param _binaryArgs - Serialized arguments containing the ChunkDelete object.
+ * @param index - (optional) The index of the file to delete of deleting multiple files.
+ * @throws If the file does not exist or if the caller is not the owner.
+ */
+export function deleteFile(_binaryArgs: StaticArray<u8>): void {
```

Comment:
Here we should use
```
  const file = args.nextSerializable<ChunkDelete>().expect('Invalid files');
```

I'm not sure this function is necessary as we could use deleteFiles with one element

Reply:
that's true, but calling this function avoids creating a for loop